protocol/shannon/fullnode_lazy.go

```diff
@@ -123,6 +132,9 @@ func (lfn *LazyFullNode) GetSession(
 			)
 	}
 
+	// Update session rollover boundaries for rollover monitoring
+	lfn.updateSessionValues(*session)
```

Comment:
`updateSessionRollverBoundaries` seems like a better name (or any other name that indicates what the method actually does)